vespa-feed-client/src/main/java/ai/vespa/feed/client/ApacheCluster.java

```diff
@@ -33,6 +33,13 @@
 class ApacheCluster implements Cluster {
 
     private final List<Endpoint> endpoints = new ArrayList<>();
+    private final List<BasicHeader> defaultHeaders = Arrays.asList(new BasicHeader("User-Agent", String.format("vespa-feed-client/%s", Vespa.VERSION)),
+                                                                   new BasicHeader("Vespa-Client-Version", Vespa.VERSION));
```

Comment:
Do we need both of these?

Reply:
I guess it's quite fast when compressed ...

Reply:
(But encodeHeaders typically has most of the perf samples for small requests ...)

Reply:
Both are needed afaik. Former for access log, latter for metric.